test/Common/FormTrait.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FrontendTest\Common;
+
+use Laminas\Form\ElementInterface;
+use Laminas\Form\FormInterface;
+use Laminas\InputFilter\Input;
+use Laminas\InputFilter\InputFilterInterface;
+
+use function count;
+
+trait FormTrait
+{
+    protected function formWillInstantiate(string $class): void
+    {
+        $this->assertInstanceOf($class, new $class());
+        $this->assertInstanceOf($class, new $class(null, []));
+        $this->assertInstanceOf($class, new $class('accountForm'));
+        $this->assertInstanceOf($class, new $class('accountForm', []));

```

Comment:
```suggestion
        $this->assertInstanceOf($class, new $class('form', []));
```
(This was something I also forgot to change when I refactored DotKernel Admin - it's fixed now)